docker/ubuntu-2004-builder.Dockerfile

```diff
@@ -123,6 +123,7 @@ RUN cd /root \
     && ./rustup.sh -y --no-modify-path \
     && /opt/cargo/bin/rustup target add wasm32-wasi \
     && /opt/cargo/bin/cargo install \
+        cargo-edit          \
```

Comment:
Where would cargo edit be used? Does this belong in the build image or in the psibase-contributor image?

Reply:
It's used to add commands to cargo to let us set the crate versions from the CLI, and I've written a script to automatically update all the rust crates but haven't PRed it yet.
Hmm, I guess it should just be added to the contributor, not necessary if one is only trying to build.